pandas/tests/io/formats/test_to_csv.py

```diff
@@ -563,3 +563,21 @@ def test_to_csv_compression(self, compression_only,
             result = pd.read_csv(path, index_col=0,
                                  compression=read_compression)
             tm.assert_frame_equal(result, df)
+
+
+def test_csv_formatter_line_terminator_default(monkeypatch):
+    # see GH #23608
+    # ensure default line_terminator used is os.linesep
+    from pandas.io.formats.csvs import CSVFormatter
+    df = DataFrame()
+
+    with monkeypatch.context() as m:
+        # fake default os.linesep to look like windows
```

Comment:
Is there a reason why this needs to look like windows? Just wondering if it wouldn't be better to just change the patched object to something ridiculous like 'foo' so that the test results won't vary by platform

Reply:
I thought the same thing initially. The reason I used this particular example is because the only os this change will affect is windows. I think the reason this bug has been around so long is probably because most devs are using mac or linux, so I thought it might be good to use windows as an example.

Reply:
I get what you are trying to emulate but I'd still think it's better served not mimicking any platform